bin/internal/update_dart_sdk.ps1

```diff
@@ -20,6 +20,7 @@ $cachePath = "$flutterRoot\bin\cache"
 $dartSdkPath = "$cachePath\dart-sdk"
 $engineStamp = "$cachePath\engine-dart-sdk.stamp"
 $engineVersion = (Get-Content "$flutterRoot\bin\internal\engine.version")
+$engineRealm = (Get-Content "$flutterRoot\bin\internal\engine.realm")
```

Comment:
So when I delete `engine.realm` on Windows, I hit:

```
Downloading Dart SDK from Flutter engine ...
Get-Content : Cannot find path 'D:\git\flutter\bin\internal\engine.realm' because it does not exist.
At D:\git\flutter\bin\internal\update_dart_sdk.ps1:23 char:17
+ $engineRealm = (Get-Content "$flutterRoot\bin\internal\engine.realm")
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\git\flutter\bin\internal\engine.realm:String) [Get-Content], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand
```

Let me see if I can make a patch for this...

Reply:
So, the idea is that the file would always be there, but when its empty, we'd use the default value. Are you wondering if maybe we shouldn't check in an empty file, and instead only include the file when we want a different realm?

Reply:
Oh, I'm sorry, I misread your other response. This is fine then :)